10_Examples/jett-onboarding/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>jett-onboarding</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="jettvanbrocklin@todo.todo">jettvanbrocklin</maintainer>
+  <license>TODO: License declaration</license>
+
```

Comment:
Doesnt matter for this onboarding, but generally its good to fill in description and license as well when you add new package for the first time (because otherwise you are never gonna go back and fill them in lol).